store/build.xml

```diff
@@ -393,6 +393,14 @@
        <arg line="-o ${dist.dir}"/>
     </java>
   </target>
+  <target name="create-version-ldap" depends="build-init" description="Creates ldap schema version: zimbra-attrs-schema">
+    <exec executable="git" failonerror="false" output="${dist.dir}/zimbra-attrs-schema">
```

Comment:
I would still expect this to go under somewhere like conf/attrs/ under ${dist.dir} - may need to create that directory if it isn't there already.